free-programming-books.md

```diff
@@ -357,6 +357,15 @@
 * [Practical and Theoretical Aspects of Compiler Construction](http://web.stanford.edu/class/archive/cs/cs143/cs143.1128/) (class lectures and slides)
 
 
+####Computer Science
+
+* [Computer Science I - Draft](https://bitbucket.org/chrisbourke/computersciencei/raw/44fb9b39be3221dc02c1b5d0712f9b9f03260e46/ComputerScienceOne.pdf) - Dr. Chris Bourke (PDF)
+* [Computer Science II - Draft](http://cse.unl.edu/~cbourke/ComputerScienceTwo.pdf) - Dr. Chris Bourke (PDF)
```

Comment:
OT: Sorry but in my previous post i pressed "Resolve conversation" and can't see the conversation anymore so i put here my amend
In his repo exists also the latest pdf version. but that's not matter because is demonstrated that he updates the book's version regularly in the  url i already committed too 

Now about that request i added   (:construction: *in process*) in CS I  II III in my branch so we can resolve that conv  too 
OT: By the way very nice that feature to cope with any issue step by step; work as an awesome reminder